src/ImageViewer.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<widget id="com.mendix.widget.custom.imageviewer.ImageViewer" needsEntityContext="true" offlineCapable="true" xmlns="http://www.mendix.com/widget/1.0/">
+    <name>Image viewer</name>
+    <description>Change a number value using a slider</description>
+    <icon/>
+    <properties>
+        <property key="source" type="enumeration" defaultValue="imageUrlAttribute">
+            <caption>Data source</caption>
+            <category>Data source</category>
+            <description></description>
+            <enumerationValues>
+                <enumerationValue key="systemImage">System image</enumerationValue>
+                <enumerationValue key="imageUrlAttribute">Dynamic URL</enumerationValue>
+                <enumerationValue key="staticUrl">Static URL</enumerationValue>
+                <enumerationValue key="staticImage">Static Image</enumerationValue>
+            </enumerationValues>
+        </property>
+        <property key="systemImageAttribute" type="attribute" required="false" isPath="optional" pathType="reference">
+            <caption>System image</caption>
+            <category>Data source</category>
+            <description>Holds the url of an image</description> 
+            <attributeTypes>
+                <attributeType name="String"/> 
+            </attributeTypes>
+        </property>
+        <property key="imageAttribute" type="attribute" required="false" isPath="optional" pathType="reference">
+            <caption>Dynamic URL</caption>
```

Comment:
Attribute captions must reflect that it's an attribute. e.g Dynamic URL attribute...